src/styles/global.ts

```diff
@@ -1088,6 +1089,30 @@ const GlobalStyles = createGlobalStyle`
       }
     }
   }
+  
+  // Heavier specification of the selector to ensure override from global.ts
+  // Ideally take out the modali default injected styles and only load from our end.
+  body.modali-open .modali.modali-size-large {
+    width: 80vw;
+    max-width: initial;
+    min-width: initial;
+    max-height: 100%;
+      @media (min-width: 500px) {
```

Comment:
can we make this bad boy an `em`

Reply:
I'll do it in my branch

Reply:
Actually, I did an exact override as using 50rem didn't seem to override the native 500px styles. Correct me if I'm wrong though.